pkg/cmd/cli/cmd/env.go

```diff
@@ -38,18 +39,19 @@ syntax.`
   // List the environment variables defined on all pods
   $ %[1]s env pods --all --list
 
-  // Output a YAML object with updated enviroment for deployment 'registry'
+  // Output a YAML object with updated environment for deployment 'registry'
   // Does not alter the object on the server
   $ %[1]s env dc/registry STORAGE_DIR=/local -o yaml
```

Comment:
change the sample path name so there is no confusion that the name "local" has anything to do with it not hitting the server